### ISSUE TITLE
NPE in Windows using v0.0.2

### ISSUE THREAD

**Issue by noodnik2** [noodnik2/flightvisualizer#9]
**Describe the bug**
NPE

**To Reproduce**
Invoke this command which returns many results:

**Expected behavior**
Nicer error, no NPE.

**Environment, Version**
Windows 10, v0.0.2

**Screenshots**
```
C:\bin>fviz tracks --tailNumber N6605D --saveArtifacts --cutoffTime 2023-06-10T08:30:00-07:00
2023/06/11 20:18:52 INFO: requesting from endpoint(/flights/N6605D?&end=2023-06-10T08:30:00-07:00)
2023/06/11 20:18:53 INFO: saving to file(fvf_N6605D_cutoff-20230610T083000-0700.json)
2023/06/11 20:18:53 INFO: requesting from endpoint(/flights/N6605D-1686406703-sw-672p/track)
2023/06/11 20:18:53 INFO: saving to file(fvt_N6605D-1686406703-sw-672p.json)
2023/06/11 20:18:53 INFO: requesting from endpoint(/flights/N6605D-1686328644-sw-2553p/track)
2023/06/11 20:18:53 INFO: saving to file(fvt_N6605D-1686328644-sw-2553p.json)
2023/06/11 20:18:53 INFO: requesting from endpoint(/flights/N6605D-1686328627-sw-102p/track)
2023/06/11 20:18:54 INFO: saving to file(fvt_N6605D-1686328627-sw-102p.json)
2023/06/11 20:18:54 INFO: requesting from endpoint(/flights/N6605D-1686328583-sw-2439p/track)
2023/06/11 20:18:54 INFO: saving to file(fvt_N6605D-1686328583-sw-2439p.json)
2023/06/11 20:18:54 INFO: requesting from endpoint(/flights/N6605D-1686328583-sw-2439p/track)
2023/06/11 20:18:54 INFO: saving to file(fvt_N6605D-1686328583-sw-2439p.json)
2023/06/11 20:18:54 INFO: requesting from endpoint(/flights/N6605D-1686268175-adhoc-2049p/track)
2023/06/11 20:18:54 INFO: saving to file(fvt_N6605D-1686268175-adhoc-2049p.json)
2023/06/11 20:18:54 INFO: requesting from endpoint(/flights/N6605D-1686264815-adhoc-70p/track)
2023/06/11 20:18:55 INFO: saving to file(fvt_N6605D-1686264815-adhoc-70p.json)
2023/06/11 20:18:55 INFO: requesting from endpoint(/flights/N6605D-1686254954-adhoc-854p/track)
2023/06/11 20:18:55 INFO: saving to file(fvt_N6605D-1686254954-adhoc-854p.json)
2023/06/11 20:18:55 INFO: requesting from endpoint(/flights/N6605D-1686196384-adhoc-1049p/track)
2023/06/11 20:18:55 INFO: saving to file(fvt_N6605D-1686196384-adhoc-1049p.json)
2023/06/11 20:18:55 INFO: requesting from endpoint(/flights/N6605D-1686193979-adhoc-1694p/track)
2023/06/11 20:18:55 INFO: requesting from endpoint(/flights/N6605D-1686191308-adhoc-6842p/track)
2023/06/11 20:18:55 INFO: requesting from endpoint(/flights/N6605D-1686193100-adhoc-1520p/track)
2023/06/11 20:18:56 INFO: requesting from endpoint(/flights/N6605D-1686188651-adhoc-2300p/track)
2023/06/11 20:18:56 INFO: requesting from endpoint(/flights/N6605D-1686171714-adhoc-1528p/track)
2023/06/11 20:18:56 INFO: requesting from endpoint(/flights/N6605D-1686072379-adhoc-2533p/track)
2023/06/11 20:18:56 INFO: couldn't get for /flights/N6605D-1686193979-adhoc-1694p/track: couldn't get successful response for https://aeroapi.flightaware.com/aeroapi/flights/N6605D-1686193979-adhoc-1694p/track: statusCode(429), status(429 Too Many Requests), body(
        {"title": "Rate limit error", "reason": "RATE_LIMIT_ERROR", "detail": "User has reached quota limit", "status": 429}
    )
2023/06/11 20:18:56 INFO: couldn't get for /flights/N6605D-1686191308-adhoc-6842p/track: couldn't get successful response for https://aeroapi.flightaware.com/aeroapi/flights/N6605D-1686191308-adhoc-6842p/track: statusCode(429), status(429 Too Many Requests), body(
        {"title": "Rate limit error", "reason": "RATE_LIMIT_ERROR", "detail": "User has reached quota limit", "status": 429}
    )
...
2023/06/11 20:18:56 INFO: couldn't get for /flights/N6605D-1686072379-adhoc-2533p/track: couldn't get successful response for https://aeroapi.flightaware.com/aeroapi/flights/N6605D-1686072379-adhoc-2533p/track: statusCode(429), status(429 Too Many Requests), body(
        {"title": "Rate limit error", "reason": "RATE_LIMIT_ERROR", "detail": "User has reached quota limit", "status": 429}
    )
2023/06/11 20:18:56 INFO: writing 9 camera-path-vector KML document(s)
2023/06/11 20:18:56 INFO: saving to file(fvk_N6605D_230610150311Z-4017Z_camera-path-vector.kmz)
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x10ad79a]

goroutine 1 [running]:
github.com/noodnik2/flightvisualizer/internal.TracksCommandArgs.saveKmlTracks({{{0xc0000be600, 0x27}, {0xc0000a6e6c, 0x1}, 0x1, {0xc0000be630, 0x20}}, 0x0, 0x0, 0x1, ...}, ...)
        /home/runner/work/flightvisualizer/flightvisualizer/internal/generatetracks.go:101 +0x1fa
github.com/noodnik2/flightvisualizer/internal.TracksCommandArgs.GenerateTracks({{{0xc0000be600, 0x27}, {0xc0000a6e6c, 0x1}, 0x1, {0xc0000be630, 0x20}}, 0x0, 0x0, 0x1, ...})
        /home/runner/work/flightvisualizer/flightvisualizer/internal/generatetracks.go:72 +0x165
github.com/noodnik2/flightvisualizer/cmd.glob..func4(0x14a2d00, {0x116cb31?, 0x5?, 0x5?})
        /home/runner/work/flightvisualizer/flightvisualizer/cmd/tracks.go:59 +0x218
github.com/spf13/cobra.(*Command).execute(0x14a2d00, {0xc0000d6230, 0x5, 0x5})
        /home/runner/work/flightvisualizer/flightvisualizer/vendor/github.com/spf13/cobra/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x14a2460)
        /home/runner/work/flightvisualizer/flightvisualizer/vendor/github.com/spf13/cobra/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/work/flightvisualizer/flightvisualizer/vendor/github.com/spf13/cobra/command.go:992
github.com/noodnik2/flightvisualizer/cmd.Execute()
        /home/runner/work/flightvisualizer/flightvisualizer/cmd/root.go:47 +0x25
main.main()
        /home/runner/work/flightvisualizer/flightvisualizer/main.go:6 +0x17

C:\Users\mross\personal\bin>
```

**Comment by noodnik2** [comment]
Should be fixed now; [merged to `main`](https://github.com/noodnik2/flightvisualizer/commit/4e056d71738c4875986c249f3f37e0efe315757b), will go out with next release.

**Comment by noodnik2** [comment]
Fixed in release [v0.0.3](https://github.com/noodnik2/flightvisualizer/releases/tag/v0.0.3).
